### ISSUE TITLE
Bootstrap button class .btn-success fails 508 color contrast 

### ISSUE THREAD

**Issue by nickbristow** [18F/18f.gsa.gov#59]
The green "Sign Up for More Information" uses the .btn-success class from bootstrap which has, 
background - #5cb85c
Text - white

This fails 508 color contrast at 2.48:1 

The hover state has,
background - #47a447
Text - White

Which fails 508 at 3.15:1

Fix Suggestion
Override bootstrap class with new background colors
Normal state
Background - 387f38

Hover
Background - 2a632a
